spec/classes/ferm_spec.rb

```diff
@@ -22,6 +22,9 @@
         it { is_expected.to contain_file('/etc/ferm.d/chains') }
         it { is_expected.not_to contain_service('ferm') }
         it { is_expected.not_to contain_file('/etc/ferm.conf') }
+        if facts[:os]['family'] == 'RedHat' and facts[:os]['release']['major'].to_i <= 6
```

Comment:
@kBite Please use ```&&``` instead of ```and``` to fix the RuboCop issue.

Reply:
@dhoppe Done. Please see my updated commits.

Reply:
Looks good. Thank you very much for your contribution.